Bookish/Bookish.DataAccess/AddBook.cs

```diff
@@ -8,10 +8,10 @@ namespace Bookish.DataAccess
 {
     public class AddBook
     {
-        public static string AddToBooks(string title, string genre, int numberOfCopies, string isbn)
+        public static string AddToBooks(string title, string genre, int numberOfCopies, string isbn, string authorID)
```

Comment:
Probably *slightly* neater to have this be an int, and use the int.Parse when calling this method.